### ISSUE TITLE
Unlink currently selected item in collection contents (content inventory) tree

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#1393]
The collection inventory tree in the Collection contents section of the collection sidebar has an accessibility issue that I think we can solve and improve the user experience around it at the same time. We currently indicate which item in the collection hierarchy is currently selected by adding a yellow highlight background:

<img width="402" alt="Screen Shot 2023-01-20 at 4 08 10 PM" src="https://user-images.githubusercontent.com/101482/213821529-0bcdfd7a-7ed6-4a10-bfee-e35727cc87e7.png">

However, the default Bootstrap blue link color does not have sufficient contrast against the yellow background to meet accessibility standards. 

At the same time, I think it's deceiving, and potentially confusing, to link the selected item. It's already selected, so we shouldn't suggest to the user that they might want to select it again (doing so just refreshes the page to the same state).

I think we can solve both of those issues by removing the link from the currently selected component. If it's not a link, it 1) won't have the unnecessary link styling and 2) the text will be displayed in the default body text color, which has sufficient contrast against the yellow highlight background to pass the accessibility standard.

After removing the link from the currently selected component, the tree should look like this:

<img width="402" alt="Screen Shot 2023-01-20 at 4 07 30 PM" src="https://user-images.githubusercontent.com/101482/213822196-842698fc-8d63-4e58-aed7-e3d4d9a30424.png">

**Comment by lfarrell** [comment]
Fix PR: https://github.com/projectblacklight/arclight/pull/1469
